deploy/application/java/java-war.md

```diff
@@ -57,16 +97,15 @@ The supported containers are listed below:
 
 ## Configure your Java application
 
-<div class="alert alert-hot-problems">
-<h4>**Warning!**</h4>
-You **must** provide a `clevercloud/war.json` file in your application repository.
-</div>
+{{< alert "warning" >}}
+<p>You **must** provide a `clevercloud/war.json` file in your application repository.</p>
```

Comment:
Same here, better use <strong> to replace `**` maybe and <code> for the backticks